tide_media.install

```diff
@@ -381,3 +354,14 @@ function tide_media_update_8009() {
dLXur9kHx072vS9975rTm034VLuYuha1obfofC0iABd370kAb2AAfkAjywSnQCZ6BGYgekoRMIX9oL7rfb0BvYSwsDpvDkXAx3IV6nBnRQcKQY8g9ZBWjgAnEVGGeY1mw27GHsI9x7DhPXC3uK14Nn4V/QhAhJBAGaIRokmmGiPLEo8QFEoU0QKtB20AnQldNL0h/ikGW4RqjNeMEUwqZj9zLHMTCyNLKSmEjsbWw+3KwcPRxpnKpcM1zt/Gk85rysfG9578mUCwYs8VJSFNYRIRdlF6MII6TwEkStzJu45QSkVaQMZJ1kdsr362IKJkrF6tMqRmo12jitaK1J3Td9Z4YWBveMTYw6TJTN2+1VLXqsDG0fW2X4SDh+Ng51VXU7b4HdQfLziaKvfcP3wp/q0BMUG9IbphnhGoUZzQcMx03npCdJJ88mnok3Wo3Y8arzMtZhft3Z8fnxuXtPIQ9XH/EpZBQdK0ksUzrGPH4WGVP9fkTJ07WnN53VqLubn14A7mx5aLnZUxTfbNjK2g7d9X1Gk3n1esRPSK9N29631q5XdIvd7fvnuv9D4MxD5cepz9ZeZry7OfziBfvXnmMPHxjNFo/Dk+Yv82fHHxP+mA4Ff2x6lP359dfvn9dnQbTK9+/zLyevTpX/CN4Xn5+8WfTr4AFroXuxZ2Ly7/Ll4SW6peVlq+tyK+cXOVYzV3zf7S/gvz64wOi1QcA+3p1dVYUAHwBAMuHV1d/16yuLp9Ak41XAFwP3fhvZ/1ZwwjAsZk11C85tOvf/7H8F6gywQU17nfKAAAACXBIWXMAAAsTAAALEwEAmpwYAAAEImlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyIKICAgICAgICAgICAgeG1sbnM6ZXhpZj0iaHR0cDovL25zLmFkb2JlLmNvbS9leGlmLzEuMC8iCiAgICAgICAgICAgIHhtbG5zOmRjPSJodHRwOi8vcHVybC5vcmcvZGMvZWxlbWVudHMvMS4xLyIKICAgICAgICAgICAgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIj4KICAgICAgICAgPHRpZmY6UmVzb2x1dGlvblVuaXQ+MjwvdGlmZjpSZXNvbHV0aW9uVW5pdD4KICAgICAgICAgPHRpZmY6Q29tcHJlc3Npb24+NTwvdGlmZjpDb21wcmVzc2lvbj4KICAgICAgICAgPHRpZmY6WFJlc29sdXRpb24+NzI8L3RpZmY6WFJlc29sdXRpb24+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgICAgIDx0aWZmOllSZXNvbHV0aW9uPjcyPC90aWZmOllSZXNvbHV0aW9uPgogICAgICAgICA8ZXhpZjpQaXhlbFhEaW1lbnNpb24+MTg8L2V4aWY6UGl4ZWxYRGltZW5zaW9uPgogICAgICAgICA8ZXhpZjpDb2xvclNwYWNlPjE8L2V4aWY6Q29sb3JTcGFjZT4KICAgICAgICAgPGV4aWY6UGl4ZWxZRGltZW5zaW9uPjE4PC9leGlmOlBpeGVsWURpbWVuc2lvbj4KICAgICAgICAgPGRjOnN1YmplY3Q+CiAgICAgICAgICAgIDxyZGY6U2VxLz4KICAgICAgICAgPC9kYzpzdWJqZWN0PgogICAgICAgICA8eG1wOk1vZGlmeURhdGU+MjAxNToxMToxMCAxMjoxMTo0NDwveG1wOk1vZGlmeURhdGU+CiAgICAgICAgIDx4bXA6Q3JlYXRvclRvb2w+UGl4ZWxtYXRvciAzLjQ8L3htcDpDcmVhdG9yVG9vbD4KICAgICAgPC9yZGY6RGVzY3JpcHRpb24+CiAgIDwvcmRmOlJERj4KPC94OnhtcG1ldGE+CsT0zU0AAAIVSURBVDgRjVNNS2JhFPbeq04t/BpoEZOBJWIkrVwYs8gfMLgwSAh3AxYxEbQJWiVBi3ZBILNtZikym1m7VnAxQ5SgG1cuxK8SB7/neS694o3rrQPH93w853nPPe9RMb0hPp9vzQ5ptVptI6hslIxEIrbxeJyWJOlnOBxeMMIqRkmLxbI3GAyOQLTa7/f/1Gq1h3n4uR0lEglLr9dLiEIQHTMm/NenUUe73W73FAUSiyaTibvZbN5Xq1XdrqZEAEqNRmMJNSvBYDBSr9ev0YWdJC8imc3mz6FQqOlwOLrRaFTO5XL/ksmkmpa8Xu+m1Wr9hllsyLLsHY1GdpDaoIJAc2JeJuizoihPeIgy5viIC2/NMLYxi0OiQaIp0nN4AS8CiQ2En1C7g0YKMlq7w5qk9IreE3M6nal0On2nDrJUKn2IxWI37Xb7YN4nvSblJ2JW3wuFwgnsnkpEEAgUv99/MRwOz2HPXQtiUTjG4K+KxeIFbHUe0wIGMpnMJWam+7wkEEIMsYKE8SkRnXw+L2OIb06cGGJZI2S6Rwxg4ZY7nc4ZgIsCoHdiTazlcvlHpVJ5EnkNKwDrWAGnSKJ1E2Kq0hZCDLHC56khQjdrKFBjLIT+crvdX6i0GaPglIlVnZcfDRE+7SOfHwv21+Vy7cfj8d1sNvubSpsx5oghdpZIY3s8nq1AIPAV/3KHJjHjMEcMsTNh038wIuU96QNsXAAAAABJRU5ErkJggg==');
```

Comment:
Is there no other way to do this? Is it just a base 64 encode?

Reply:
Yeah it's a base64 encoded icon.

Is it that it **is** a base64 encoded icon that you don't like or the method of updating the value?

You question does make me realise that the image file is still part of the repo, which I'll push a new commit to remove.

Reply:
Ah, it's a minor thing... I was hoping there would be something like...

```
$media_config->set('icon.data', base64_encode(file_get_contents('star.png'));
```
 But it still doesn't do anything for cleaning the string out of the YAML. Apparently you can write your own loaders for Symfony YAML but we won't be doing that :)